EIPS/eip-pbmrc1.md

```diff
@@ -10,64 +10,84 @@ created: 2023-04-01
+  - PBM Tokens are bearer instruments, with self-contained programming logic, and can be transferred between two parties without involving intermediaries. It combines the concept of:
+    - programmable payment - automatic execution of payments once a pre-defined set of conditions are met; and
+    - programmable money - the possibility of embedding rules within the medium of exchange itself that defines or constraints its usage.
+- **PBM Creator** defines the conditions of the PBM Wrapper to create PBM Tokens. A PBM Creator is able to issue any amount of PBM Tokens, provided that the PBM Creator deposits equivalent amounts of fiat currencies with the Spot Token Issuer.
+- **PBM Infrastructure** - consisting of a ledger-based infrastructure. While a PBM can be either distributed ledger technology (DLT) based or non-DLT based, the scope of this PBMRC paper is limited to a DLT-based infrastructure build upon the Ethereum blockchain;
+- **PBM Wallet** - cryptographic wallets which holds users' private keys, granting them access to PBMs.
 
 ## Specification
-The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119 and RFC 8174.
-
-### Overview 
-- Whether a PBM **SHOULD** have an expiry time will be decided by the PBM creator, the spec itself should not enforce an expiry time.
-    - In lieu of our goals of making PBM a suitable construct for all kinds of business logic that could occur in the real world.
-    - Should an expiry time not be needed, the expiry time can be set to infinity.
-- PBM **MUST** provide a mechanism for all transacting parties to verify the condition by which the token of value can be unwrapped
-- PBM **MUST** wrap an underlying token of value.
-    - The wrapping of the token can be done either upon the creation of the PBM or wrapped on the fly at a later date.
-    - A token of value can be implemented in any widely accepted ERC. E.g. ERC-20, ERC-777, ERC-1363
-        - The definition of the word value goes beyond the current scope of our work - refer to project orchid white paper page 14 for details.
-    - Consequently, the semantic use of the words "wrap" and "unwrap" would convey the bounding and unbounding of the underlying token.
-- PBM **SHALL** adhere to the definition of “wrapping” or “wrap” to mean bounding a token in accordance with PBM business logic during its lifecycle stage.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [BCP 14](https://www.rfc-editor.org/info/bcp14) [ [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt) ] [ [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) ] when, and only when, they appear in all capitals, as shown here.
```

Comment:
oh yar are there any other portions of the writing in which i use the word “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and “OPTIONAL”

but didnt capitalized it?